integreat_cms/cms/models/abstract_tree_node.py

```diff
@@ -282,7 +287,19 @@ def move(self, target, pos=None):
         # Moving a node can modify all other nodes via raw sql queries (which are not recognized by cachalot),
         # so we have to invalidate the whole model manually.
         invalidate_model(self.__class__)
-        super().move(target=target, pos=pos)
+        try:
+            with db_mutex("page-tree"):
+                super().move(target, pos)
+        except DBMutexError as e:
+            logger.error("Could not retrieve database mutex to save %s", self)
```

Comment:
```suggestion
            logger.error("Could not retrieve database mutex to save %r", self)
```
For debugging purposes, I always find it more helpful to get the page id etc. in the log instead of the title